colorbleed/__init__.py

```diff
@@ -1,6 +1,7 @@
 import os
 from pyblish import api as pyblish
-from avalon import api, pipeline
+
+from .config_actions import register_actions
```

Comment:
If you refactor the `register_actions` to `register_launcher_actions` then you don't need the additional function here at lines 22-24. Then the import is just there to expose it to the launcher. :) Better?

Reply:
Not sure why this shows as outdated as I commented on line 4. :confused:

Reply:
Nice catch! Will make things a bit clearer :)